src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -122,6 +122,10 @@ export class ProsemirrorAdapter {
         ];
     }
 
+    public disconnectedCallback() {
+        this.view.destroy();
```

Comment:
Ah. I just realised that we need to re-initialise the editor in `connectedCallback`, in case someone disconnects it, and then connects it again (typically if you move the editor around in the DOM). We never do that ourselves, but since it's lime-elements, we need to cover such use-cases anyway.

However, I think we can push that off until after the MVP, so I'll just create an issue for it now.